pkg/reader/ccip.go

```diff
@@ -628,6 +628,10 @@ func (r *ccipChainReader) discoverDestinationContracts(
 	}
 	for chain, cfg := range sourceConfigs {
```

Comment:
Discussed offline: maybe sort these to ensure deterministic ordering across oracles, so that the first router grab is the same for all nodes.